content/en/docs/tasks/observability/metrics/customize-metrics/test.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1090,SC2154,SC2155,SC2034
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source "tests/util/samples.sh"
+
+# @setup profile=default
+
+## Setting up application
+# Set to known setting of sidecar injection
+kubectl label namespace default istio-injection=enabled --overwrite
+
+# Install Bookinfo application
+startup_bookinfo_sample
+
+## Before patching configuration
+# First make sure that the metric dimensions we are going to add as
+# part of this task don't exist yet
+function send_productpage_requests() {
+  _set_ingress_environment_variables
+  local GATEWAY_URL="$INGRESS_HOST:$INGRESS_PORT"
+  for _ in {1..100}; do
+    snip_verify_the_results_1 > /dev/null
+  done
+}
+
+send_productpage_requests
+_verify_not_contains snip_verify_the_results_2 "destination_port"
+_verify_not_contains snip_verify_the_results_2 "request_host"
+
+echo "$snip_enable_custom_metrics_2" | istioctl install --set tag="$TAG" --set hub="$HUB" -f -
```

Comment:
Why is --set tag and hub needed?

Reply:
I assume it would be set in Prow. Right? 

The reason why I added them - I work with my private registry (which also resides in my local machine) where I push Istio images (created from `make docker` in istio). Pulling over the internet (from gcr) is a pain. Main reason is bandwidth constraints, internet connection issues and timeouts over here. As I saw the test results, it does not cause issues in prow as well[?]  In my local environment I use `IfNotPresent` pull policy to reduce time while running `make docs.test`.